app/scripts/proactive/view/LogoutView.js

```diff
@@ -52,6 +52,11 @@ define(
                 function tryToConnect() {
                     if (StudioClient.isLoggedIn()) {
                         isConnected()
+                    } else {
+                        var connectionError = $('<div id="connection-error" class="btn-group pa-tools"><label style="color:red; font-weight:bold; font-size:15px; padding-left:4px; padding-top:4px;">You have been disconnected from the server. Reload the page in order to log into the portal again. </label></div>');
```

Comment:
why adding this in logout view ?

Reply:
this is where the check is done if the user is still logged in:
 if (StudioClient.isLoggedIn()) {

Reply:
yes, this means that you will wait for the second check:
isLoggedIn returns localStorage['pa.session'] != null
so before removing the session id it doesn't mean you are still connected.

Reply:
where do you think this should be added?